src/main/java/com/sequenceiq/cloudbreak/controller/json/UpdateClusterJson.java

```diff
@@ -8,6 +8,7 @@
 
     private HostGroupAdjustmentJson hostGroupAdjustment;
     private StatusRequest status;
+    private ReinstallJson reinstallJson;
```

Comment:
so the json would look like this?
```
{
  "reinstallJson": {
    "blueprintId": xxx,
    "hostGroups": []
  }
}
```
I think blueprintId and hostgroups can go directly in the updateJson